app/components/project-card-skills.js

```diff
@@ -1,21 +1,73 @@
 import Ember from 'ember';
 
-export default Ember.Component.extend({
+const {
+  Component,
+  set,
+} = Ember;
+
+/**
+  `project-card-skills` composes the given project's list of skills on the
+  project's card.
+
+  ## default usage
+
+  ```Handlebars
+  {{project-card-skills skills=project.skills}}
+  ```
+
+  @class project-card-skills
+  @module Component
+  @extends Ember.Component
+ */
+export default Component.extend({
   classNames: ['project-card-skills'],
+
+  /**
+    Returns whether or not the overflow skills are hidden.
```

Comment:
We're showing both skills they have and skills they don't have. Skills they have come first.